pages/draft-07/json-hyper-schema-release-notes.md

```diff
@@ -12,56 +9,56 @@ so keywords used for other purposes (`readOnly` and `media`) have been
 
 The [new draft](../../draft-07/json-schema-hypermedia.html) has been completely rewritten
 for clarity and accessibility, so it is best to simply approach it as a new
-proposal.  We hope to add tutorial material at some point, but that is
+proposal. We hope to add tutorial material at some point, but that is
```

Comment:
suggestion: remove the sentence "we hope to add tutorial material..." as it doesn't add valuable information for the reader.

Reply:
Done.